### DISCUSSION TITLE
[Wiki] - Screenshoots

### DISCUSSION THREAD

**Discussion by blackcrack** [valinet/ExplorerPatcher#1053] [General]
Hi,

a bit more over your nice Programm would be nice,  
because 2-3 or 4  Pictures Say's more as 1000 Words,  
because couple Peoples use some nice program not until they have see a screenshot  
and know how it works and where goes on the Direction,  
so would it very nice if in the Wiki a couple Screenshots and a bit information about the Pictures..  
  
and Vali,  
thank you for this very nice Proggy/Patcher.. i loooo'fffhhe it :)  
  
best  
Blacky

**Comment by Poopooracoocoo** [comment]
Where specifically would you add screenshots btw?
